src/components/Holdings.tsx

```diff
@@ -78,24 +79,30 @@ export default function Holdings() {
       >
         <div css={rootStyle}>
           <Heading title="CELO" gridArea="celo" />
-          <Amount
-            iconSrc={"/assets/tokens/CELO.svg"}
-            context="Funds frozen in on-chain Reserve contract"
-            loading={isLoadingCelo}
-            label="Frozen"
-            units={celo.frozen.units}
-            value={celo.frozen.value}
-            gridArea="frozen"
-          />
+          {celo.frozen.value > 0 ? (
+            <Amount
+              iconSrc={"/assets/tokens/CELO.svg"}
+              context="Funds frozen in on-chain Reserve contract"
+              loading={isLoadingCelo}
+              label="Frozen"
+              units={celo.frozen.units}
+              value={celo.frozen.value}
+              gridArea="frozen"
+            />
+          ) : (
+            <div css={hiddenCelo}></div>
+          )}
+
           <Amount
             iconSrc={"/assets/tokens/CELO.svg"}
             context="Funds in on-chain Reserve contract and in custody"
             loading={isLoadingCelo}
-            label="Unfrozen"
```

Comment:
make this s ternary so that when there is more than 0 unfrozen it says unfrozen

Reply:
so that it says CELO when zero and unfrozen otherwise

Reply:
🔥🔥🔥 Good Idea!